test/Utils/TupleReader.hpp

```diff
@@ -174,7 +174,7 @@ decodeUserTuple(BUFFER &buf, Data<BUFFER> &data)
 	std::vector<UserTuple> results;
```

Comment:
What about `UserTuple::field3` which is a double and is also not initialized by default initializer? In this sense, it is not clear what you are fixing (would be nice if you would attach the build error to your commit message).

Why didn't you go for explicitly initializing the struct fields with default values rather than initializing? I.e.:
```diff
diff --git a/test/Utils/TupleReader.hpp b/test/Utils/TupleReader.hpp
index 040726c..d0078e3 100644
--- a/test/Utils/TupleReader.hpp
+++ b/test/Utils/TupleReader.hpp
@@ -36,9 +36,9 @@

 /** Corresponds to data stored in _space[512]. */
 struct UserTuple {
-       uint64_t field1;
+       uint64_t field1 = {};
        std::string field2;
-       double field3;
+       double field3 = {};
        void *field4 = reinterpret_cast<void *>(0xDEADBEEF);
        std::pair<std::string, uint64_t> field5;
 };
```

Reply:
It doesn't make any difference for me. Rewrote it in your way.

Reply:
Just so you understand what I meant about the compiler error: see commit message of tarantool/tntcxx@2aaba882e, for instance.